executor/executor.go

```diff
@@ -288,6 +289,64 @@ type ShowDDLJobsExec struct {
 	jobs   []*model.Job
 }
 
+// ShowDDLJobQueriesExec represent a show DDL job queries executor.
+type ShowDDLJobQueriesExec struct {
+	baseExecutor
+
+	cursor int
+	jobs   []*model.Job
+	jobIDs []int64
+	query  string
+}
+
+// Open implements the Executor Open interface.
+func (e *ShowDDLJobQueriesExec) Open(ctx context.Context) error {
+	if err := e.baseExecutor.Open(ctx); err != nil {
+		return errors.Trace(err)
+	}
+	jobs, err := admin.GetDDLJobs(e.ctx.Txn())
+	if err != nil {
+		return errors.Trace(err)
+	}
+	historyJobs, err := admin.GetHistoryDDLJobs(e.ctx.Txn())
```

Comment:
If you get the corresponding job from `GetDDLJobs`, maybe you needn't to `GetHistoryDDLJobs`.
`GetHistoryDDLJobs` only return 10 jobs, I think it can't meet your needs. You need to adjust the number of jobs.

Reply:
GetHistoryDDLJobs only return 10 jobs,this understanding,It is in 10 historical jobs search.

Reply:
I think if the user gives the job ID, they want to get the corresponding query. So I think `GetHistoryDDLJobs` only return 10 jobs that cannot meet the demand.

Reply:
@zimulala If user give job ID exactly in the 10 history jobs, then show the corespoding query is ok

Reply:
@winkyao But if a user gets the job ID from the log, and it isn't in 10 history jobs.  
And, generally speaking, I gave the job ID, then it should return the corresponding query.